### PR TITLE
add PythonAnywhere & Glitch

### PR DESCRIPTION
**Pull request by dx0naclyoo**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1944,5 +1944,7 @@ HTML Themes and UI Kits to create your applications faster!
   * [SimpleRestore](https://simplerestore.io) - Hassle-free MySQL backup restoration. Restore MySQL backups to any remote database without code or a server.
   * [360Converter](https://www.360converter.com/) - Free tier useful website to convert: Video to Text && Audio to Text && Speech to Text && Real-time Audio to Text && YouTube Video to Text && add Video Subtitle. Maybe it will be helpful in a short video conversion or in a short youtube tutorial:)
   * [QRCodeBest](https://qrcode.best/) - Create custom QR codes with 13 templates, full privacy, and personal branding. Features tracking pixels, project categorization, and unlimited team seats on QRCode.Best.
+  * [PythonAnywhere](https://www.pythonanywhere.com/) - PythonAnywhere makes it easy to create and run Python programs in the cloud. You can write your programs in a web-based editor or just run a console session
```
[the remark this answers is not in the document]
Comment (dx0naclyoo) [reply README.md]: Corrected

**Comment by ripienaar** [comment]
thanks

**Comment by imba-tjd** [comment]
pythonanywhere has a whitelist for out traffic: https://www.pythonanywhere.com/whitelist/
It was famous ten years ago, but is really unattractive for now.

**Comment by ripienaar** [comment]
Though tbf, its the only way a service like this remains free and avoid the bitcoin miners

**Comment by imba-tjd** [comment]
Besides, both PythonAnywhere & Glitch are already exitst in the list :joy:

**Pull request merged**